godot/prefabs/GameController/GameController.cs

```diff
@@ -0,0 +1,80 @@
+using Godot;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public partial class GameController : Node
+{
+	public Passenger[] AvailablePassengers { get; private set; } = Array.Empty<Passenger>();
+	private Turtle turtleInstance;
+	private Timer gameTimer;
+	
+	[Export]
+	public float Score { get; private set; } = 0;
+
+	public override void _Process(double delta)
+	{
+		// Increment score every second
+		Score += 1f;
```

Comment:
_Process here will run once every 1/60th (frame rate). You'll need to multiply this by delta to get an accurate value (or just pass thru delta)